src/cyclotom.c

```diff
@@ -688,7 +688,9 @@ Obj             Cyclotomic (
             res[0] = cof;
         else {
             CHANGED_BAG( ResultCyc );
-            res[0] = DIFF( INTOBJ_INT(0), cof );
+            Obj negcof = DIFF( INTOBJ_INT(0), cof );
```

Comment:
I am happy to merge this, though you might want to change this to `Obj negcof = AInvInt(cof);`.

Reply:
No I can't, because cof can be a rational :) (seperate note, some int related functions could produce more helpful errors when given non integres)

Reply:
Ah, sorry for the red herring

Reply:
(Actually, I have a déjà-vu ... didn't we discuss the exact same thing at the same spot before?)